mission/functions/core/ui/taskroster/fn_tr_getMapPosClick.sqf

```diff
@@ -19,18 +19,20 @@
     Example(s): none
 */
 
+params ["_control", "_button", "_xPos", "_yPos", "_shift", "_ctrl", "_alt"];
```

Comment:
`params` ==> documents function arguments rather than relying on humans to magically know what elements are in `_this`